requirements/constraints.txt

```diff
@@ -77,5 +77,5 @@ jsonfield2==3.0.3
 # django-storages==1.9 breaks contentstore imports
 django-storages==1.8
 
-# edx-when==0.7.0 has a breaking change
-edx-when==0.6.0
```

Comment:
You'll notice this doesn't actually change any requirements down the line. That's because they were already on 0.7.0, from a manual upgrade that missed removing this constraint line.

Reply:
Oh, yeah, I totally missed that that line had been added when I merged my 0.7.0 change (which I guess I hadn't rebased and then re-run `make upgrade` on?)